src/OmniSharp.Roslyn.CSharp/Services/Refactoring/V2/BaseCodeActionService.cs

```diff
@@ -51,6 +55,41 @@ protected BaseCodeActionService(OmniSharpWorkspace workspace, CodeActionHelper h
             }
         }
 
+        private void SetOrderedProviders()
+        {
+            try
+            {
```

Comment:
Why is this in a try/catch? What are you expecting to fail?

Reply:
the GetSortedCodeFixProviders() will return an exception if there are cycles in the graph. This try catch will handle the exception and return the unordered list if that exception occurs